lib/improver/tests/ensemble_calibration/ensemble_calibration/test_EstimateCoefficientsForEnsembleCalibration.py

```diff
@@ -832,11 +836,13 @@ def test_current_forecast_cubes_len_zero_catch_warning(self,
 
         plugin.estimate_coefficients_for_ngr(
             current_forecast, historic_forecasts, truth)
+        warning_msg = "Insufficient input data present to estimate "
```

Comment:
I think this warning message can be changed to:
```python
warning_msg = ("Insufficient input data present to estimate "
                           "coefficients using NGR.")
```
This is so the second line is included within the test of the warning message.